addons/im_livechat/static/tests/composer_patch_tests.js

```diff
@@ -109,10 +110,13 @@ QUnit.test('display canned response suggestions on typing ":"', async (assert) =
         source: "hello",
         substitution: "Hello! How are you?",
     });
-    const { openDiscuss } = await start();
+    const { advanceTime, openDiscuss } = await start({ hasTimeControl: true });
     await openDiscuss(channelId);
     assert.containsNone($, ".o-mail-Composer-suggestionList .o-open");
     await insertText(".o-mail-Composer-input", ":");
+    await advanceTime(DEBOUNCE_FETCH_SUGGESTION_TIME);
+    await nextTick();
+    await nextTick();
```

Comment:
`waitUntil` instead?

Reply:
Indeed can be a bit cleaning with `waitUntil`. Might do that later, better merge it quickly to fix race condition in tests asap